test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -615,7 +615,7 @@ protected bool CheckEqualityFromPathsCore(string relPath, string basePath, strin
 
         private static void GetNumbersFromFile(ref string firstString, ref string secondString, decimal precision)
         {
-            Regex _matchNumer = new Regex(@"\b[0-9]+\.?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+            Regex _matchNumer = new Regex(@"\b[0-9]+[\.,]?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Are there any other decimal separators in other languages that we should care about?.  I think using CurrentCulture.NumberFormatInfo was a good idea.